### ISSUE TITLE
Initialization for a style guide

### ISSUE THREAD

**Issue by varunachar** [aritraroy/Flashbar#5]
Would recommend having a way of initializing the library once by passing a config object and then the same Flashbar object be used. When you pass the config object, it creates an immutable version of a `Flashbar` which gets cloned every time you try to customize it further.

This basically allows someone to initialize the Flashbar once based on their own style guide and then keep reusing that configuration. 

Varun

**Comment by aritraroy** [comment]
This makes sense. Devs who would want to use the same style guide multiple times can build something called a `FlashbarConfig` once and reuse it to create multiple `Flashbar` instances.

Or this problem can also be solved by exposing a new `clone()` API, that lets you clone an existing `Flashbar` instance and build a completely new one with the existing configs.

**Comment by varunachar** [comment]
Clone will require you to hold a copy of an existing Flashbar instance until you need to recreate it. Best to create a new one everytime using a config.

**Comment by aritraroy** [comment]
Yes, makes sense. Will prioritize this for the upcoming release.
